.github/FUNDING.yml

```diff
@@ -0,0 +1,2 @@
+github: [RehanSaeed,henrygab]
```

Comment:
I would like to exclude myself from any funding.

Reply:
Done.

Reply:
Wouldn't that require removal of my alias from the FUNDING.yml?